### PR TITLE
Added named entities call and test.

### PR DESCRIPTION
**Pull request by anneCarlson**
Going to write up a guide for adding this to other languages, also.

### CODE REVIEW

File: spec/indico_spec.rb [hunk 1]
```diff
@@ -100,11 +100,28 @@
     expect Set.new(response.keys).subset?(Set.new(expected_keys))
   end
 
-  it 'should tag text with correct keywords' do
-    expected_keys = Set.new(%w(guns kill people))
-    response = Indico.keywords('Guns don\'t kill people. People kill people.')
+  # add test when keyword api is added to public cloud
```
Comment (sihrc) [inline spec/indico_spec.rb]: Key words are on the public cloud now i believe.

Comment (anneCarlson) [inline spec/indico_spec.rb]: The tests were failing, I'll recheck why that was.

File: spec/indico_batch_spec.rb [hunk 2]
```diff
@@ -108,14 +108,36 @@
...
+    response = Indico.batch_named_entities(['I want to move to New York City!'])
+    expect(response.length).to eql(1)
+    response = response[0]
+
+    expect(response.keys.length > 0).to eql(true)
+    expect(response.values[0]['confidence'] >= 0.75).to eql(true)
+    expect(Set.new(response.values[0]['categories'].keys)).to eql(expected_keys)
+
+    chance_sum = response.values[0]['categories'].values.inject{|sum,x| sum + x }
+    expect(chance_sum > 0.9999).to eql(true)
```
Comment (sihrc) [inline spec/indico_batch_spec.rb]: we could do stuff like `expect(target).to be < value` also. Don't know if it matters too much

File: spec/indico_batch_spec.rb [hunk 3]
```diff
@@ -108,14 +108,36 @@
...
+
+    config = { threshold: 1 }
```
Comment (sihrc) [inline spec/indico_batch_spec.rb]: maybe should be in a different test?

File: spec/indico_spec.rb (hunk 1 above)
Comment (anneCarlson) [inline spec/indico_spec.rb]: Passing now. Did it go up in the last 2 days?

File: spec/indico_batch_spec.rb (hunk 2 above)
Comment (anneCarlson) [inline spec/indico_batch_spec.rb]: Ah, yes, that's much nicer. I'll update it.

File: spec/indico_spec.rb (hunk 1 above)
Comment (sihrc) [inline spec/indico_spec.rb]: Probably :P

**Comment by sihrc** [comment]
:+1: Looks good to me!

**Comment by madisonmay** [comment]
LGTM as well.  I'll rebase and merge this in...

**Comment by anneCarlson** [comment]
I'm purposefully not merging until tomorrow, sinc eI link to this an an example for the other library updattes in my outline.

**Comment by madisonmay** [comment]
Haha, ok, fair.

**Pull request merged**